physics/geopotential_body.hpp

```diff
@@ -50,17 +51,17 @@ struct Geopotential<Frame>::Precomputations {
   Vector<Inverse<Length>, Frame> grad_𝔏_vector;
 
   // These quantities depend on n but are independent from m.
-  FixedVector<Inverse<Length>, size> ℜ;  // 0 unused.
+  FixedVector<Inverse<Length>, size> ℜ{uninitialized};  // 0 unused.
```

Comment:
Here uninitialized does nothing unless you add an uninitialized constructor to Quantity.

We should do that.

I could do that in a separate pull request; add a TODO.

Reply:
#1958